lib/ansible/modules/network/aci/aci_interface_policy_leaf_policy_group.py

```diff
@@ -0,0 +1,399 @@
+   - Choice of monitoring_policy to be used as part of the leaf policy group to be created.
+   aliases: [ monitoring_policy, monitoring_policy_name ]
+ storm_control_interface_policy:
+   description:
+   - Choice of storm_control_interface_policy to be used as part of the leaf policy group to be created.
+   aliases: [ storm_control_interface_policy, storm_control_interface_policy_name ]
+ l2_interface_policy:
+   description:
+   - Choice of l2_interface_policy to be used as part of the leaf policy group to be created.
+   aliases: [ l2_interface_policy, l2_interface_policy_name ]
+ port_security_policy:
+   description:
+   - Choice of port_security_policy to be used as part of the leaf policy group to be created.
+   aliases: [ port_security_policy, port_security_policy_name ]
+ attached_entity_profile:
+   description:
+   - Choice of attached_entity_profile (AEP) to be used as part of the leaf policy group to be created.
+   aliases: [ attached_entity_profile, attached_entity_profile_name, AEP, AEP_name ]
+ state:
+   description:
+   - Use C(present) or C(absent) for adding or removing.
+   - Use C(query) for listing an object or multiple objects.
+   choices: [ absent, present, query ]
+   default: present
+'''
+
+EXAMPLES = r'''
+- name: creating a Port Channel (PC) Interface Policy Group
+  aci_interface_policy_leaf_policy_group:
+    hostname: apic
+    username: yourusername
+    password: yourpassword
+    policy_group: policygroupname
+    description: policygroupname description
+    link_aggregation_type: link
+    link_level_policy: whateverlinklevelpolicy
+    fibre_channel_interface_policy: whateverfcpolicy
+    state: present
+
+- name: creating a Virtual Port Channel (VPC) Interface Policy Group
+  aci_interface_policy_leaf_policy_group:
+    hostname: apic
+    username: yourusername
+    password: yourpassword
+    policy_group: policygroupname
+    description: policygroupname description
+    link_aggregation_type: node
+    link_level_policy: whateverlinklevelpolicy
+    fibre_channel_interface_policy: whateverfcpolicy
+    state: present
+
+- name: creating a Leaf Access Port Policy Group
+  aci_interface_policy_leaf_policy_group:
+    hostname: apic
+    username: yourusername
+    password: yourpassword
+    policy_group: policygroupname
+    description: policygroupname description
+    link_aggregation_type: leaf
+    link_level_policy: whateverlinklevelpolicy
+    fibre_channel_interface_policy: whateverfcpolicy
+    state: present
+
+- name: deleting an Interface policy Leaf Policy Group
+  aci_interface_policy_leaf_policy_group:
+    hostname: apic
+    username: yourusername
+    password: yourpassword
+    policy_group: policygroupname
+    link_aggregation_type: type_name
+    state: absent
+'''
+
+RETURN = ''' # '''
+
+from ansible.module_utils.network.aci.aci import ACIModule, aci_argument_spec
+from ansible.module_utils.basic import AnsibleModule
+
+
+def main():
+    argument_spec = aci_argument_spec
```

Comment:
`aci_argument_spec` has been updated to be a function.  This line should be updated as:

```
argument_spec = aci_argument_spec()
```

Reply:
alrighty ! 👍